src/main/java/hudson/plugins/jira/JiraCreateIssueNotifier.java

```diff
@@ -50,6 +50,9 @@
     private String testDescription;
     private String assignee;
     private String component;
+    private String priority;
+    private String type;
+    private Integer actionIdOnSuccess;
 
     enum finishedStatuses {
```

Comment:
Drop down list, probably not, maybe override in csv of finished statuses? These are checked by taking the string value of status and then equals ignore case against the enum. Unfortunately I don't have the development time to implement it at this stage and I think it would be better suited in a follow up pull request anyway.